Tools/Scripts/delete-if-too-large

```diff
@@ -0,0 +1,25 @@
+import sys
+import os
+from pathlib import Path
+
+
+def main():
+    directory = sys.argv[1]
+    directoryPath = Path(directory).resolve()
+
+    if not directoryPath.exists():
+        print(directory + "does not exist")
+        return 1
+
+    if not directoryPath.is_dir():
+        print(directory + "is not a directory")
+        return 1
+
+    tenGigabytes = 1024 * 1024 * 1024 * 10
+    if directoryPath.stat().st_size >= tenGigabytes:
+        os.remove(directory)
+        print("Successfully removed " + directory)
```

Comment:
Maybe better to say: f"Deleted {directory}" (instead of using the word successfully)

Reply:
Done.